publisher/kafka.go

```diff
@@ -78,8 +79,8 @@ func (pr *Kafka) ProduceBulk(events []*pb.Event, connGroup string, deliveryChann
 		m := d.(*kafka.Message)
 		if m.TopicPartition.Error != nil {
 			eventType := events[i].Type
-			metrics.Decrement("kafka_messages_delivered_total", fmt.Sprintf("success=true,conn_group=%s,event_type=%s", connGroup, eventType))
-			metrics.Increment("kafka_messages_delivered_total", fmt.Sprintf("success=false,conn_group=%s,event_type=%s", connGroup, eventType))
+			metrics.Increment("kafka_messages_undelivered_total", map[string]string{"success": "true", "conn_group": connGroup, "event_type": eventType})
```

Comment:
Correct, but for calculating the difference, we need common labels so that it can be group on hence kept the same as the preemptive increment of `kafka_messages_delivered_total`